--- perl-5.28.0/Configure.orig	2018-05-21 15:29:23.000000000 +0300
+++ perl-5.28.0/Configure	2018-11-19 13:45:03.650885176 +0200
@@ -1427,8 +1427,7 @@
 i_whoami=''
 : Possible local include directories to search.
 : Set locincpth to "" in a hint file to defeat local include searches.
-locincpth="/usr/local/include /opt/local/include /usr/gnu/include"
-locincpth="$locincpth /opt/gnu/include /usr/GNU/include /opt/GNU/include"
+locincpth=""
 :
 : no include file wanted by default
 inclwanted=''
@@ -1444,12 +1443,11 @@
 : change the next line if compiling for Xenix/286 on Xenix/386
 xlibpth='/usr/lib/386 /lib/386'
 : Possible local library directories to search.
-loclibpth="/usr/local/lib /opt/local/lib /usr/gnu/lib"
-loclibpth="$loclibpth /opt/gnu/lib /usr/GNU/lib /opt/GNU/lib"
+loclibpth=""
 
 : general looking path for locating libraries
 glibpth="/lib /usr/lib $xlibpth"
-glibpth="$glibpth /usr/ccs/lib /usr/ucblib /usr/local/lib"
+glibpth="$glibpth /usr/ccs/lib /usr/ucblib"
 test -f /usr/shlib/libc.so && glibpth="/usr/shlib $glibpth"
 test -f /shlib/libc.so     && glibpth="/shlib $glibpth"
 test -d /usr/lib64         && glibpth="$glibpth /lib64 /usr/lib64 /usr/local/lib64"
@@ -5034,7 +5032,7 @@
 : Set private lib path
 case "$plibpth" in
 '') if ./mips; then
-	plibpth="$incpath/usr/lib $sysroot/usr/local/lib $sysroot/usr/ccs/lib"
+	plibpth="$incpath/usr/lib $sysroot/usr/ccs/lib"
     fi;;
 esac
 case "$libpth" in
@@ -21422,7 +21420,7 @@
 	case "$ranlib" in
 	:) ranlib='';;
 	'')
-		ranlib=`./loc ranlib X /usr/bin /bin /usr/local/bin`
+		ranlib=`./loc ranlib X /usr/bin /bin`
 		$test -f $ranlib || ranlib=''
 		;;
 	esac
